pom.xml

```diff
@@ -1149,17 +1149,16 @@
                 <artifactId>commons-codec</artifactId>
                 <version>1.10</version>
             </dependency>
-            <dependency>
-                <groupId>commons-collections</groupId>
-                <artifactId>commons-collections</artifactId>
-                <version>3.2.2</version>
-                <!-- <version>3.1</version> xmlui - wing -->
-            </dependency>
             <dependency>
                 <groupId>org.apache.commons</groupId>
                 <artifactId>commons-collections4</artifactId>
                 <version>4.1</version>
             </dependency>
+            <dependency>
+                <groupId>commons-collections</groupId>
+                <artifactId>commons-collections</artifactId>
+                <version>3.2.2</version>
```

Comment:
This dependency should be dropped, correct?

Reply:
No, we still have the issue with `beanutils` mentioned above. Removing that dependency results in:

```
[WARNING] 
Dependency convergence error for commons-collections:commons-collections:3.2.1 paths to dependency are:
+-org.dspace:dspace-api:7.0-SNAPSHOT
  +-commons-validator:commons-validator:1.5.0
    +-commons-beanutils:commons-beanutils:1.9.2
      +-commons-collections:commons-collections:3.2.1
and
+-org.dspace:dspace-api:7.0-SNAPSHOT
  +-commons-validator:commons-validator:1.5.0
    +-commons-collections:commons-collections:3.2.2

[WARNING] Rule 0: org.apache.maven.plugins.enforcer.DependencyConvergence failed with message:
Failed while enforcing releasability the error(s) are [
Dependency convergence error for commons-collections:commons-collections:3.2.1 paths to dependency are:
+-org.dspace:dspace-api:7.0-SNAPSHOT
  +-commons-validator:commons-validator:1.5.0
    +-commons-beanutils:commons-beanutils:1.9.2
      +-commons-collections:commons-collections:3.2.1
and
+-org.dspace:dspace-api:7.0-SNAPSHOT
  +-commons-validator:commons-validator:1.5.0
    +-commons-collections:commons-collections:3.2.2
]
```

Reply:
@tomdesair : Aha, yes, I neglected to reread the discussion above.  You are correct, we do need it still.